nemo/collections/tts/losses/fastpitchloss.py

```diff
@@ -120,6 +120,37 @@ def forward(self, pitch_predicted, pitch_tgt, len):
         return pitch_loss
 
 
+class EnergyLoss(Loss):
+    def __init__(self, loss_scale=0.1):
+        super().__init__()
+        self.loss_scale = loss_scale
+
+    @property
+    def input_types(self):
+        return {
+            "energy_predicted": NeuralType(('B', 'T'), RegressionValuesType()),
+            "energy_tgt": NeuralType(('B', 'T'), RegressionValuesType()),
+            "len": NeuralType(('B'), LengthsType()),
+        }
+
+    @property
+    def output_types(self):
+        return {
+            "loss": NeuralType(elements_type=LossType()),
+        }
+
+    @typecheck()
+    def forward(self, energy_predicted, energy_tgt, len):
+        if energy_tgt is None:
+            return 0.0
```

Comment:
nit: `forward` function is expected to return a Tensor. Should we enforce a Tensor here as well? @ryanleary @redoctopus ?